astropy/wcs/wcsapi/high_level_api.py

```diff
@@ -117,116 +117,196 @@ def world_to_array_index(self, *world_objects):
             return tuple(_toindex(self.world_to_pixel(*world_objects)[::-1]).tolist())
 
 
-class HighLevelWCSMixin(BaseHighLevelWCS):
+def high_level_objects_to_values(*world_objects, low_level_wcs):
```

Comment:
These need to be added to ``__all__`` right?

Reply:
Do they? Are these functions we need to propagate up into the `astropy.wcs.wcsapi` namespace? or are we ok with them living in here?